### DISCUSSION TITLE
WIN + X doesn´t work

### DISCUSSION THREAD

**Discussion by payback666** [valinet/ExplorerPatcher#173] [Q&A] [answered]
Pressing Win + X gets me a restart of the taskbar but no GUI, new Version installed and works on .282

**Comment by Gaurav-Original-ClassicShellTester** [comment, marked as answer]
You mean on your Windows 11 22000.282, you got the classic Taskbar and everything working but Win+X is crashing Explorer.exe and restarting it ? Is there any other hotkey app that could be conflicting with it?

**Reply by payback666** [reply]
Yes, that´s what I mean.  No other hotkey app installed.

**Reply by payback666** [reply]
Installed new and now it works as it should.

**Comment by techlogik** [comment]
I found that after following the uninstall procedure, which worked. It isn't possible to reinstall by following the procedures. Drop the file into c:\windows...restart explorer and nothing happens, no appdata folder recreated, nothing download???? Not possible I guess?

**Comment by nonKurau** [comment]
WIN + X doesn't work with latest releases (i'm on 22000.376.40.18 now) when Taskbar style = Windows 10 (with win11 default option it works just fine). Nothing happens when WIN + X pressed. Right clicking start button also does nothing. I'm on 22523.1000 dev version of Windows 11, so it might be the problem. Anyway just reporting.

**Reply by valinet** [reply]
It works. The problem is that on newer builds, you need symbols for things to work. If symbols are not available/downloaded, then some functionality is disabled. Please consult: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

Also, EP is not really supported on Insider builds, as it is pretty cumbersome to fully support it there. Also, Win+X with Windows 11 taskbar has nothing to do with EP, it’s hosted and managed properly directly by Windows.

**Reply by nonKurau** [reply]
Oh, thanks, now I see. Enabled console and it really says: 
>Failure in reading symbols for "C:\WINDOWS\system32\twinui.pcshell.dll"

Luckily most of the functionality I need is on, so I still can use EP with success even on dev builds. Thank you very much for your magnificent work!
